lib/app/models/testing_instance.rb

```diff
@@ -281,6 +275,40 @@ def token_expiration_time
         token_retrieved_at + token_expires_in.seconds
       end
 
+      def get_sequence_requirement(requirement)
+        return unless requirement&.dig(:name)
+
+        attributes = { name: requirement[:name],
```

Comment:
Don't need to set `name: requirement[:name]` if we're then merging in `requirement`.